recipes/bertrand/all/conandata.yml

```diff
@@ -2,3 +2,6 @@ sources:
   0.0.12:
     url: https://github.com/bernedom/bertrand/archive/0.0.12.tar.gz
     sha256: 0648edbb9ea39a918f718a672e14b26c614d8409f146c1550becd45d881cfcf4
+  0.0.14:
+    url: https://github.com/bernedom/bertrand/archive/0.0.14.tar.gz
+    sha256: f6ba82e715f9371f06a0d0bd6b7f017c3ff0709e8e5142d8fd4ba00aec1228ba
```

Comment:
```suggestion
    sha256: f6ba82e715f9371f06a0d0bd6b7f017c3ff0709e8e5142d8fd4ba00aec1228ba

```

Reply:
Reasons why my PRs for conan-center-index fail:

[] recipe not working
[] conanfile.py is not styled with PEP8
[] code is not compiling
[x] forgot the newline at the end of conandata.yml 

(BTW: it's fixe now :))